test/lib/processors/nonAsciiEscaper.js

```diff
@@ -0,0 +1,101 @@
+const test = require("ava");
```

Comment:
I'm missing a test for `latin1` encoded strings

Reply:
done